app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -60,6 +62,18 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def self.seed
+    embedded_in_zone ||
+      create_with(:url => "https://localhost:8443", :zone => Zone.default_zone)
```

Comment:
So I think this zone should by `MiqServer.my_server.my_zone`. That way the provider's work will follow the server that is running the role.